### ISSUE TITLE
Laminography multiply factor to align the projection image to slice image

### ISSUE THREAD

**Issue by hws203** [LLNL/LEAP#76]
@kylechampley   When I make any normal slice from leap package's lamino-option, I have to add some factor which multiply to sdd value by 20. And also I have to divide the voxelwidth and voxelheight by 20. If I remove this 20 multiply then the slice looks normal only at the small center area.  That is too clumsy and it is different from other recon package.

Could you explain why this happens in mathematical point ?

1. with 20 multiply:

<img width="794" alt="with_20times_factor" src="https://github.com/LLNL/LEAP/assets/73321363/2b1b406e-cc5f-404c-9375-1053e36b30ed">

<img width="783" alt="with_20times_factor2" src="https://github.com/LLNL/LEAP/assets/73321363/1b0a4506-3eb7-4ea6-b172-e003dec5ff78">

2. without 20 multiply:
<img width="787" alt="without_20times_factor" src="https://github.com/LLNL/LEAP/assets/73321363/aad34e31-8cd6-4b1c-93b9-dc3e7b508d45">

<img width="779" alt="without_20times_factor2" src="https://github.com/LLNL/LEAP/assets/73321363/822c843a-1929-416f-b478-e32a5c427fad">

**Comment by kylechampley** [comment]
Sorry, I don't understand your explanation.

**Comment by hws203** [comment]
Below is my test script, please check the red circle. I have to add some multiply factor(20) to get normal wide area aligned slice from lami-recon.

<img width="893" alt="oct_script" src="https://github.com/LLNL/LEAP/assets/73321363/e9766c64-64aa-44d5-af22-8bc17101c316">

**Comment by hws203** [comment]
I guess that this issue may be related to the tilt of detector angle, so I have tried to align it for several days, but I can not match it until now without this 20 scale factor.

**Comment by kylechampley** [comment]
Well it makes sense that if your sdd is 20 times larger, that your reconstruction voxels would have to be 20 times smaller.  This is the concept of "magnification factor in cone-beam CT.

Consider a ray going from the source to a particular detector pixel and another ray going from the source to the adjacent detector pixel, then at the center of rotation these rays will be separated by sod/sdd*pixelWidth which is your native voxel size.  Thus you can see from this equation that if sdd is 20 times larger, then your voxels should be 20 times smaller.

Now I don't know why you had to multiply your sdd by 20.  Could it really be this far off?  I am wondering if your detector pixel size is correct.  Everything should be in units of mm; maybe your detector pixel size was listed in um (micrometers)?

**Comment by hws203** [comment]
Oh Yes it is the wrong pixel size setting, after changing it to 0.00748 from 0.0748. I can see the same normal recon slice without this 20 scale factor. But I wonder how this same setting can be covered by my other recon package.

**Comment by hws203** [comment]
I can see that your print_parameters() shows new mean sod and sdd value with some calculation and those value are longer than the origin setting, so I guess that your algorithm may need to make a new pixel size too at the case of laminography. 
how about your thought?

**Comment by kylechampley** [comment]
Ya, I don't know what to say about that.  Apparently LEAP works when you give it the correct detector pixel size, so I don't know what I can really do beyond that.

I'm always happy to help explain how LEAP works or fix any bugs that the users find, but there is a limit to how much I can help troubleshoot a specific dataset; I am offering this help free of charge and in my "free" time.

**Comment by hws203** [comment]
If this issue consumes your time too much, then I will use current solution.  I just guess that you can fix it easily with short time at your own free time. I will close this issue.

**Comment by hws203** [comment]
I fine tuned the tilt of detector and I can see almost same slice as like one after ct-magnification factor. so it is definitely related to my own sample case. I am sorry to border you.
